server/game/cards/02_SHD/events/Outflank.ts

```diff
@@ -0,0 +1,29 @@
+import { EventCard } from '../../../core/card/EventCard';
+import { RelativePlayer, WildcardCardType, WildcardLocation } from '../../../core/Constants';
+import AbilityHelper from '../../../AbilityHelper';
+
+export default class Outflank extends EventCard {
+    protected override getImplementationId () {
+        return {
+            id: '0802973415',
+            internalName: 'outflank',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.setEventAbility({
+            title: 'Attack with two units (one at a time)',
+            initiateAttack: {
```

Comment:
Just for clarity let's do this as the below instead of using an empty `initiateAttack`. They are both equivalent in function and the form below is a little easier to understand than an empty `initiateAttack` imo.

```typescript
immediateEffect: AbilityHelper.immediateEffects.attack()
```

The attack inside of `then` can stay the way it is

Reply:
done